recipe/meta.yaml

```diff
@@ -44,7 +43,6 @@ test:
     - huggingface_hub >= 0.12.1
     - setuptools-rust >= 1.5.2
     - pytest >= 7.2.0
-    - pytest-benchmark >= 4.0.0
     - hypothesis >= 6.70.2
     - pytorch >= 1.10
```

Comment:
Could you remove whitespaces before numbers for all cases in `run_constrained` and `test/requires`?
Instead of `hypothesis >= 6.70.2` should be `hypothesis >=6.70.2`